ibm_catalog.json

```diff
@@ -462,7 +462,7 @@
                             {
                                 "diagram": {
                                     "caption": "Secure infrastructure on VPC with ROKS",
-                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone/main/patterns/diagrams/svgs/roks/roks.drawio.svg",
+                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone/terraform-ibm-landing-zone/patterns/diagrams/svgs/roks/roks.drawio.svg",
```

Comment:
This is not correct - Ill fix in https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/pull/297

Reply:
Thanks for catching both, @ocofaigh. Sorry, moving too fast.